CMakeLists.txt

```diff
@@ -780,16 +781,6 @@ if(VITA)
         -DVITA -DNO_SCREENSHOT -DTHEXTECH_FIXED_RES -DFIXED_RES -DLOW_MEM -DTHEXTECH_PRELOAD_LEVELS -DNO_INTPTROC)
 endif()
 
-if(NINTENDO_WII)
-    # Link Wii specific libraries to the generic interface.
-    target_link_libraries(A2XT_Int INTERFACE db)
-    target_link_libraries(A2XT_Int INTERFACE wiiuse)
-    target_link_libraries(A2XT_Int INTERFACE fat)
```

Comment:
These three are essential to the game overall (even if we did a soundless build) so I'd rather keep them in the main CMake, the others are sound-related and can definitely go to MixerX.